openssl/src/dh.rs

```diff
@@ -1,3 +1,17 @@
+//! dh.rs
+//! 
+//! summary: bindings for dh library of OpenSSL
+//! author: steven fackler
+//! version: 
+//!
+//! dh is the diffie-hellman (dh) algorithm which allows for two parties to derive the key used for
+//! communication. in dh, the two parties agree to some large prime, `p`, and a generator, `g`,
+//! where 0 < g < p.
+//!
+//! For a full summary of the algo, please read https://wiki.openssl.org/index.php/Diffie_Hellman
```

Comment:
Can the links be shortened up instead of being raw URLs in the documentation? I.e. `For a full summary of Diffie-Hellman, refer to the [OpenSSL wiki](https://wiki.openssl.org/index.php/Diffie_Hellman).`

Reply:
+1, didn't even know this was possible in rustdoc